deep_quoridor/test/quoridor_env_test.py

```diff
@@ -0,0 +1,201 @@
+import pytest
+
+from deep_quoridor.src.quoridor_env import QuoridorEnv
+
+
+def parse_board(board):
+    """
+    Parse a string representation of a Quoridor board (of any size) and convert it into a QuoridorEnv instance.
```

Comment:
This now became a serialization format? :rofl:  you never know how standards get started